app/requirements.txt

```diff
@@ -29,13 +29,15 @@ model-mommy==1.6.0
 psycopg2-binary==2.7.7
 pyexcel==0.5.14
 pyexcel-xlsx==0.5.7
+pyjwt>=1.7.1
 python-dateutil==2.7.3
+python-jose>=3.0.0
 pytz==2018.4
 requests==2.21.0
 six==1.11.0
 seqeval==0.0.6
 social-auth-app-django==3.1.0
-social-auth-core[azuread]==3.0.0
```

Comment:
Why remove the `azuread` extra dependency here? This will break users who are currently using this authorization provider.

Reply:
My bad. I (incorrectly) assumed that replacing it with `social-auth-core` would mean all the dependencies would get added including those for Azure and for OpenID. Fixing it now.